views/index.hbs

```diff
@@ -411,32 +466,96 @@
-                <h2>WHAT IF I DON'T HAVE A TEAM?</h2>
-                <p>You are free to hack alone, but that's no fun! We encourage everyone to be on teams of around 4-5 hackers. Official team signups happen at the start of the hackathon. If you do not have a team in mind, we hold a team formation session before hacking begins.</p>
-                <h2>WHAT TOOLS AND HARDWARE WILL BE PROVIDED FOR MY TEAM?</h2>
-                <p>If you would like to see a specific part or device at the hackathon, you can specify it on your application and we'll do our best to accommodate! When applications are released, we will also release a list of parts. In addition, keep in mind that you are not allowed to bring any hardware of your own to the hackathon.</p>
-                <h2>WHAT DO HACKERS GAIN FROM PARTICIPATING?</h2>
-                <p>In addition to being completely free, we will provide hackers free meals, a free shirt, a room to sleep in, and a chance to win a prize! The top three teams’ members will each receive a prize. Even more valuable, at our hackathon you will gain project experience and learn something new!</p>
-                <h2>HOW DO I CONTACT IDEA HACKS?</h2>
-                <p>Email us at <a class='faq-link' href='mailto:team@ideahacks.la'>team@ideahacks.la</a>! You can also like our Facebook page and follow us on Instagram.</p>
+
+            <div class="q-a">
+                <div class="question-section clearfix">
+                    <p class="q">Q.</p>
+                    <p class="question">What does a "Hardware Hackathon" mean?</p>
+                </div>
+                <div class="answer-section clearfix">
+                    <p class="a">A.</p>
+                    <p class="answer">At IDEA Hacks, a hardware hackathon means that your project can be anything you like as long as it includes a hardware component.</p>
+                </div>
+            </div>
+
+            <div class="q-a">
+                <div class="question-section clearfix">
+                    <p class="q">Q.</p>
+                    <p class="question">What if I don't know anything about hardware?</p>
+                </div>
+                <div class="answer-section clearfix">
+                    <p class="a">A.</p>
+                    <p class="answer">You are definitely not alone! During the hackathon, there will be workshops on hardware and hacking that will be accessible for everyone. In addition, our resources page provides quickstart info and basic ideas for specifc parts featured in our <a class='faq-link' href='/boxes'>boxes</a>. Create an account to check out what's in these boxes!</p>
```

Comment:
This one is on me too; I wasn't clear when I sent you the list of questions. They don't need to create an account to view the boxes, it's so they can view the resources page. Also, you should probably put a link to `/dashboard/resources` where you say "resources page". They'll get intercepted by the login page, but it makes it a little clearer we're talking about a page on the website

Reply:
Fixed!